srcs/driver/readline_wraper.c

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   readline_wraper.c                                  :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: dkolida <dkolida@student.42warsaw.pl>      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/07/21 19:22:49 by dkolida           #+#    #+#             */
+/*   Updated: 2024/07/21 23:11:33 by dkolida          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+char	*ft_read_line(char *line)
+{
+	if (line)
+	{
+		free (line);
+		line = (char *) NULL;
```

Comment:
Why are you casting NULL to char* here? From what I checked, the C standard guarantees that NULL can be assigned to any pointer type without an explicit cast.